packages/helm-charts/ethstats/templates/ethstats.deployment.yaml

```diff
@@ -26,19 +26,40 @@ spec:
       - name: ethstats
         image: {{ .Values.ethstats.image.repository }}:{{ .Values.ethstats.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
+        command:
+        - /bin/sh
+        - -c
+        args:
+        - |
+          cat <<'EOF' > /celostats-server/lib/utils/config.js
```

Comment:
This feels a little weird to generate a javascript file here, what do you think about either passing in the addresses as env variables here or maybe putting this into a .json file that's read by a .js file in ethstats?

Reply:
raised as https://github.com/celo-org/celo-monorepo/issues/1837